### PR TITLE
remove unused terminology

### PR DESCRIPTION
**Pull request by danwing**
Remove Discard Preference, Intentional Management, Traffic Shaping.  

Of those, only Discard Preference was used, and only in the abstract! The others 
were not used at all.

### CODE REVIEW

**Review by sridharangirish91 — APPROVED** [review]

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 1]
```diff
@@ -154,23 +154,14 @@ collaboration signals.
...
-competing packets of that same flow.
-
-Intentional Management:
-: Network policy such as (monthly) bandwidth
-quota or bandwidth limit, or quality (delay and/or jitter)) assurances.
-
 Per-Flow Metadata:
 : Refers to metadata that doesn't change often during the lifetime
-of a connection and thus can be exchanged once or as needed. This is communicated per flow (i.e., UDP 4-tuple) between client and network.
+of a connection and thus can be exchanged once or as needed. This is communicated per flow (i.e., UDP 4-tuple) between client and network.  This is sent from client to network.
```
Comment (boucadair) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
of a connection and thus can be exchanged once or as needed. This is communicated per flow (i.e., UDP 4-tuple) between client and network.
```

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 2]
```diff
@@ -154,23 +154,14 @@ collaboration signals.
...
 : Examples of such metadata are client request to honor per-packet metadata and preferences.
 
 Per-Packet Metadata:
 : Refers to metadata that varies packet to packet within the same flow, often capturing
-the nature and characteristics of the traffic each packet carries. This needs to be communicated on a per packet basis.
+the nature and characteristics of the traffic each packet carries. This needs to be communicated on a per packet basis. This is sent from server to network.
```
Comment (boucadair) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
the nature and characteristics of the traffic each packet carries. This needs to be communicated on a per packet basis.
```

**Review by boucadair — CHANGES_REQUESTED** [review]
We don't need to restrict the direction at the stage. It is not easy for a reader to digest why per-* is possible for a given direction only.

**Comment by John-Kaippallimalil** [comment]
Good to remove unused terminology. Changes look good.

**Review by boucadair — APPROVED** [review]
Thanks, Dan.

**Pull request merged**